src/itensor.jl

```diff
@@ -792,7 +792,9 @@ end
 @propagate_inbounds @inline function _getindex(T::Tensor, ivs::Vararg{<:Any,N}) where {N}
   # Tried ind.(ivs), val.(ivs) but it is slower
   p = NDTensors.getperm(inds(T), ntuple(n -> ind(@inbounds ivs[n]), Val(N)))
-  return _getindex(T, NDTensors.permute(ntuple(n -> val(@inbounds ivs[n]), Val(N)), p)...)
+  fac = NDTensors.permfactor(p, ivs...) #<fermions> possible sign
```

Comment:
I thought I had made a comment here but maybe I didn't post it.

Does this affect the performance for the non-QN version?

Reply:
For the non-QN version, this should just call a version of permfactor which is defined as a single-line function that just always returns the number +1 (without even checking a boolean flag or anything). But there's separately the practical question of timing the code to be sure of no hidden slowdowns.